src/neo/Network/P2P/Payloads/InvPayload.cs

```diff
@@ -30,7 +29,7 @@ public static IEnumerable<InvPayload> CreateGroup(InventoryType type, UInt256[]
                 yield return new InvPayload
                 {
                     Type = type,
-                    Hashes = hashes.Skip(i).Take(MaxHashesCount).ToArray()
+                    Hashes = hashes[i..(i + MaxHashesCount)]
                 };
```

Comment:
It may throw `OutOfRange` exception here.
![image](https://user-images.githubusercontent.com/43396869/69926103-1ed0e500-14ee-11ea-9ea7-9db75308a1fb.png)

Reply:
See #1314